_posts/2022-10-24-why-should-python-open-source-matter-science.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: single
+title: "Why should Python open source package health matter to scientists? (and to you!)"
+excerpt: "Free and open source software tools are the foundation for thousands if not millions of scientific workflows. Yet, it is rare that users fully understand it's importance in moving science forward. Here, I discuss the value of free and open source software for science; why you as a scientist should care; and what pyOpenSci is doing to try to support Python scientific tools for science. "
+author: "Leah Wasser"
+permalink: /blog/why-python-open-source-software-matters-for-scientists
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - blog-post
+  - highlight
+  - python-packaging
+  - peer-review
+toc: false
+---
+
+## Why Python open source package health should matter to you as a scientist 
+
+If you are a scientist, the health of a scientific python package may not be something that 
+you care about. What might seem more important is doing your science, and processing 
+your data to get to that coveted scientific inquiry and exploration stage. 
+
+In actuality, package health is incredibly important to science, especially open,
+data-intensive science. It should be important to you too! 
+
+Why? Let me provide a few reasons below:
+
+#### 1. Free and open source tools (FOSS) make your code simpler. They provide commonly needed functionality wrapped up in simple tested functions and objects that you don't need to recreate yourself.
```

Comment:
This heading might be a bit too large. I think the first sentence is great as a heading itself, and the next one can be worked into the paragraph below

Reply:
fixed!! thanks!